Algorithms/Python/insertion_sort.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from typing import TypeVar
+
+T = TypeVar("T", bound=int | float)
+
+def insertion_sort(arr):
+    
+    n = len(arr)
+    if n <= 1:
+        return arr
+    
+    """
+    Insertion sort algorithm to sort the array in ascending order.
+    """
```

Comment:
Docstring are well written, but they only considered as docstring iff the string is placed in triple quotes :white_check_mark: and just below the function defination :x: 

```py
def function():
    """This is function docstring"""
    pass
```
```py
def function():
    pass
    """This is not a function docstring"""
```